e2e/hook_test.go

```diff
@@ -142,42 +114,15 @@ func testHook() {
 			Expect(hasFinalizer).Should(Equal(true), "finalizer is not set: pvc=%s", pvc.Name)
 		}
 	})
-	It("should test hooks for inline ephemeral volumes", func() {
+
+	It("should test hooks for generic ephemeral volumes", func() {
 		if isStorageCapacity() {
 			Skip(skipMessageForStorageCapacity)
 			return
 		}
 
-		By("waiting controller pod become ready")
```

Comment:
deleted this test case because this and the following test case are duplicated:
https://github.com/topolvm/topolvm/blob/v0.12.0/e2e/suite_test.go#L124-L133